airbyte-webapp/src/components/connectorBuilder/types.ts

```diff
@@ -284,6 +367,21 @@ export const convertToManifest = (values: BuilderFormValues): PatchedConnectorMa
             config: {},
           },
         },
+        paginator: stream.paginator
+          ? {
+              type: "DefaultPaginator",
+              page_token_option: {
+                ...stream.paginator.pageTokenOption,
+                // ensures that empty field_name is not set, as connector builder server cannot accept a field_name if inject_into is set to 'path'
+                field_name: stream.paginator.pageTokenOption.field_name
+                  ? stream.paginator.pageTokenOption.field_name
+                  : undefined,
```

Comment:
This could also be solved by modifying the BuilderField to not set this value on the formik values when it is an empty string, but this was a more straightforward change. Currently, when it is an empty string, it is still setting it on the formik forms. Thoughts?

Reply:
I think it's fine to have this lightweight translation logic in here.